TriblerGUI/widgets/settingspage.py

```diff
@@ -1,4 +1,7 @@
-from PyQt5.QtWidgets import QWidget, QFileDialog
+from PyQt5 import QtGui
+from PyQt5.QtWidgets import QWidget, QLabel, QFileDialog
+from PIL.ImageQt import ImageQt
+import qrcode
```

Comment:
Just an idea, we can make this `qrcode` import conditional, something like:

```
try:
    import qrcode
    has_qr = True
except ImportError:
    has_qr = False
```

And show an error if has_qr is `False`?

Reply:
Yeah, that is probably better, so it causes less dependency errors. Do you have an example how I can call an error widget?

Reply:
Yes, see https://github.com/Tribler/tribler/blob/devel/TriblerGUI/dialogs/confirmationdialog.py#L46